### PR TITLE
Allow requests from new accounts

### LINKED ISSUE #1917: User is not able to scan QR code after the user taps on the Request button from the homepage by creating new  accounts with verified and unverified phone number.
**Frequency:** 100%

**Repro on build version:** Android Internal Release Build V1.25.1, iOS Test flight Release build V1.25.1
**Repro on devices:** iPhone 7 (13.3), iPhone 13 mini(15.1.1),Google Pixel 2XL (11.0), Samsung Galaxy A8+(8.0)

**Pre-condition:** 
1. User must have installed the app 
2. User must have completed the onboarding flow by tapping the “Create new account” button.

**Repro Steps:**
1. Tap on request button from homepage
2. Tap on the QR icon present in the top right corner in the request screen.
3. Scan the QR code and observe.

**Current Behavior:**  User is not able to scan QR code after the user taps on the Request button for verified and unverified phone number.

**Expected Behavior:** User should be able to scan QR code after the user taps on the Request button for verified and unverified phone number.

**Investigation:**
1. User is not able to scan QR code and also no error message appears.
2. User is able to send request by scanning QR code by restoring account having valid balance.
3. User is able to send request by adding address in request screen for new account created also.

**Impact:** Bad user impact as user is not able request by scanning qr code.

**Attachment:**
[Verified number.mp4](https://drive.google.com/file/d/1OqlHGbhRclKz6Pk0jpjuxPx1IIqOBnME/view?usp=sharing )
[Unverified number.mp4](https://drive.google.com/file/d/1ViJ1gnkGIw6CNo2lndg0l7pdtIcYeEby/view?usp=sharing)

### PR DESCRIPTION
**Pull request by MuckT**
### Description

A small change to allow requesting from newly created accounts. 

### Other changes

N/A

### Tested

Tested locally on Android & iOS.

### How others should test

1. Create a new account.
2. Attempt to request from another account using the QR code scanner.
3. Complete request flow.

### Related issues

- Fixes #1917

### Backwards compatibility

Yes

### CODE REVIEW

File: packages/mobile/src/send/utils.ts [hunk 1]
```diff
@@ -289,6 +289,14 @@ export function* handleSendPaymentData(
       isFromScan,
       origin: SendOrigin.AppSendFlow,
     })
+  } else if (isOutgoingPaymentRequest) {
+    navigate(Screens.SendAmount, {
```
Comment (gnardini) [inline packages/mobile/src/send/utils.ts:293]: This is the exact same line as the `else` below, right? I think if we want to do this we can add `if (!canSendTokens && ! isOutgoingPaymentRequest) { throw error } to the condition below

Comment (MuckT) [reply packages/mobile/src/send/utils.ts:293]: I had that in a pervious commit and wasn't sure if handling it in a new else if would, be more explicit. Happy to go back to what was in https://github.com/valora-inc/wallet/pull/1957/commits/f0f7228b1a5e92a713cd4645c961a66316722360.

**Review by gnardini — APPROVED** [review]
😄

**Comment by ValoraQA** [comment]
Hey @MuckT, We have verified the above issue in Latest Android Internal Release build V:1.26.0 and updated the status in test rails.

**Test rail links:**
**Android 8:** https://valoraapp.testrail.io/index.php?/tests/view/9821&group_by=cases:section_id&group_order=asc&group_id=35

**Android 10:** https://valoraapp.testrail.io/index.php?/tests/view/9947&group_by=cases:section_id&group_order=asc&group_id=35

**Devices:** Samsung Galaxy A8+(8.0), Samsung Galaxy M40(10.0)

Thanks..!!

**Comment by ValoraQA** [comment]
Hey @MuckT, We have verified the above issue in Latest Android Internal Release build V:1.26.0, Test Flight Release Build V1.26.0 and updated the status in test rails.

**Test rail links:**
**Android 11:**
https://valoraapp.testrail.io/index.php?/tests/view/9884&group_by=cases:section_id&group_order=asc&group_id=35

**iOS 14:**
https://valoraapp.testrail.io/index.php?/tests/view/9758&group_by=cases:section_id&group_order=asc&group_id=35

**Devices:** Google Pixel 2XL(11.0) , iPhone 12(14.7.1)

Thanks.!!

**Comment by ValoraQA** [comment]
Hey @MuckT, We have verified the above issue in Latest Test Flight Release Build V1.26.0 and updated the status in test rails.

**Test rail link:**
**iOS 13**
https://valoraapp.testrail.io/index.php?/tests/view/10013&group_by=cases:section_id&group_order=asc&group_id=35

**Devices:** iPhone 7(13.3)

Thanks.!!

**Pull request merged**